src/s4a_the_church_first_visit.tw2

```diff
@@ -0,0 +1,111 @@
+
+I could see the venue’s manager, Kiki, off to the side checking things off on her clipboard. If I was gonna throw a raging party or plan the perfect murder, I’d call Kiki either way.
+
+They ran a tight ship. These folks were putting on a show, but they were also guarding a throne.]]
+
+(click: "guarding a throne")[$dissolve[It was obvious they weren’t starting up quite yet. I had some time.
+
+But, there’s no harm in trying to [[find a way in early|The Church - First visit - Hub]].]]
+
+(set: $churchInvestigated to it +1)(set: $churchWindowSeen to true)
+
+::The Church - First visit - Front door
+The janky front doors, one used to be a wooden folding table and the other was the side of a port-a-potty, were chained shut. There was a show bill taped to the door.
+
+(click: "show bill")[$dissolve[//TONITE
+SURPRISE DEATH MATCH
+ALL COMERS WELCOME
+R U MOUNTAIN READY?
+“Your bill is your ticket. Your word is a weapon.”//
+
+[[I snatched the bill from the door|The Church - First visit - Hub]]. A lot of people would be here tonight and someone would know something useful.]]
+
+(set: $churchInvestigated to it +1)(set: $churchFrontDoorSeen to true)
+
+::The Church - First visit - Back door
+The back door was the oldest, rustiest part of the building. It looked pretty feeble, but when I tried the handle it wouldn’t budge.
+
+I tried rattling the door around.
+
+(click:"I tried rattling the door around")[$dissolve[I could shake the door quite a bit, so it wasn’t rusted shut, but it was definitely locked. Well… shit.
+
+[[I kicked the dumbass door|The Church - First visit - Kick back door]].
+\]]
+
+::The Church - First visit - Kick back door
+The door bounced me back and my ass fell into a cleaning bucket. I braced for the splash of juicy floor water, but I just heard the crinkling of paper.
+
+I pulled the bucket off and turned around to see what was inside.
+
+(click: "what was inside")[$dissolve[It was a note, but the handwriting was terrible. I tried squinting to make out the text, which never helps, but it did feel right. When I made out the first line, my heart sank.
+
+Someone had given my dipshit step dad a job again. God dammit. With the first line acting as an expectation-lowering rosetta stone, I deciphered the rest of [[the note|The Church - First visit - Ron letter]].
+\]]
+
+::The Church - First visit - Ron letter
+<div style="padding-left: 2em;">//Dear Ron,
+
+This is also Ron hehe what’s up good buddy?? :) Anyway, don’t forget to come unlock the door 2nite so that fukkin’ straglers can come in to the chalenge the champs 2nite.
+
+(click: "champs 2nite")[$dissolve[I no u wont buddy but the boss said rite a note. Keep this on u at all times!!!]]
+
+(click: "at all times")[$dissolve[EVEN WHEN U GO TO BAR!! Prove to the boss ug ot this one man!!
+
+Love,
+[Ron]<finalRon|]]//</div>
```

Comment:
THE FINAL RON

Reply:
Definitely had created enough variables at this point that I was starting to goof off a bit.